lib/person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_reader :name, :interests, :supplies
+
+  def initialize(info_hash)
+    @name = info_hash[:name]
+    @interests = info_hash[:interests]
+    @supplies = {}
+  end
+
+  def add_supply(item, quantity)
+    supplies[item] ||= 0
+    supplies[item] += quantity
+  end
+
+  def can_build?(craft)
+    @supplies == craft.supplies_required
+  end
```

Comment:
It is unlikely that these lists will match exactly. We should go through each `supplies_required` as a person may have more than the required supplies